backend/src/chats/chat/chat.controller.ts

```diff
@@ -7,18 +7,29 @@ import {
 	Param,
 	Delete,
 	BadRequestException,
+	Query,
 } from '@nestjs/common';
 import { ChatService } from './chat.service';
 import { CreateChatDto } from './dto/create-chat.dto';
 import { UpdateChatDto } from './dto/update-chat.dto';
 import { Chat } from './entities/chat.entity';
 import * as bcrypt from 'bcrypt';
-import {} from 'typeorm';
+import { FindOptionsOrder } from 'typeorm';
+import { ChatRelationsBodyDto } from './dto/chat-relations-body.dto';
+import { ChatRelationsQueryDto } from './dto/chat-relations-query.dto';
 
 @Controller('chats')
 export class ChatController {
 	constructor(private readonly chatService: ChatService) {}
 
+	private readonly defaultRelationships = { has_users: true };
+	private readonly defaultOrder: FindOptionsOrder<Chat> = {
+		name: 'ASC',
+		messages: {
+			created_at: 'ASC',
+		},
```

Comment:
created at ASC? me is confused (:

Reply:
this is for ordering when doing a query
so the default order for the messages is on the `created_at` column in ASCending order

Reply:
cool!